src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    public Circle(double radius, String color) {
+        super(color);
+        this.radius = radius;
```

Comment:
The `radius` variable is used but not declared in the class. You need to declare `radius` as a private double field in the `Circle` class to avoid compilation errors.